src/backend/src/controllers/projects.controllers.ts

```diff
@@ -299,3 +299,54 @@ export const editProject = async (req: Request, res: Response) => {
   // return the updated work package
   return res.status(200).json(updatedProject);
 };
+
+export const setProjectTeam = async (req: Request, res: Response) => {
+  const { body } = req;
+
+  // check for valid WBS number
+  const parsedWbs: WbsNumber = validateWBS(req.params.wbsNum);
+
+  if (!isProject(parsedWbs)) {
+    return res.status(404).json({ message: `${req.params.wbsNum} is not a valid project WBS #!` });
+  }
+
+  const wbsEle = await prisma.wBS_Element.findUnique({
+    where: {
+      wbsNumber: {
+        carNumber: parsedWbs.carNumber,
+        projectNumber: parsedWbs.projectNumber,
+        workPackageNumber: parsedWbs.workPackageNumber
+      }
+    },
+    include: {
+      project: true
+    }
+  });
+
+  if (wbsEle === null) {
+    return res.status(404).json({ message: `project ${req.params.wbsNum} not found!` });
+  }
+
+  // find the associated project
+  const projectID = wbsEle.project?.projectId;
```

Comment:
instead of doing it like this, you can find the project itself so that the project part isnt optional by doing like `prisma.project.findFirst({ where: { wbsElement: {carNumber: ..., projectNumber: ..., ... } } )` if that makes sense. then it returns the project itself

Reply:
also should be `projectId` for consistency (lowercase d)